pallets/api/src/fungibles/tests.rs

```diff
@@ -147,6 +147,30 @@ fn clear_metadata_works() {
 	});
 }
 
+#[test]
+fn mint_works() {
+	new_test_ext().execute_with(|| {
+		let amount: Balance = 100 * UNIT;
+		create_asset(ALICE, ASSET);
+		let balance_before_mint = Assets::balance(ASSET, &BOB);
+		assert_ok!(Fungibles::mint(signed(ALICE), ASSET, BOB, amount));
+		let balance_after_mint = Assets::balance(ASSET, &BOB);
+		assert_eq!(balance_after_mint, balance_before_mint + amount);
+	});
+}
+
+#[test]
+fn burn_works() {
```

Comment:
Should cover more cases like:
- if the value is saturated (like `decrease_allowance`) 
- or if burning more than the total balance of the account.
- check the `total_supply` of the asset wether it is updated.

Reply:
See above